src/Components/Agenda/AgendaItemMaintenancePane/AgendaItemMaintenancePane.jsx

```diff
@@ -484,6 +491,24 @@ const AgendaItemMaintenancePane = (props) => {
             { !TODLoading && combinedTodDropdown() }
 
           </div>
+          <div className="panel-meeting-agendas-user-info mt-20">
+            <div className="item">
+              <span className="label">Languages: </span>
+              <span>{userLanguages}</span>
+            </div>
+            <div className="item">
+              <span className="label">PP/Grade: </span>
+              {userPayPlan} {userGrade}
+            </div>
+            <div className="item">
+              <span className="label">Skill: </span>
+              {userSkill}
+            </div>
+            <div className="item">
+              <span className="label">CDO: </span>
+              {userCDOFirst} {agendaItem?.cdo?.last_name ?? ''}
```

Comment:
```suggestion
              {agendaItem?.cdo?. first_name ?? ''} {agendaItem?.cdo?.last_name ?? ''}
```

would this result in the same outcome?

Reply:
nope, the space gets removed